docs/guides/waffle-testing.md

```diff
@@ -31,7 +35,7 @@ Welcome to Hardhat v2.0.0
   Quit
 ```
 
-Select `Create a sample project`. This will create some files but also install the necessary packages.
+Select `Create a sample project`. This will create some files, install the `hardhat-ethers` and `hardhat-waffle` plugins, and other necessary packages.
```

Comment:
```suggestion
Select `Create a sample project`. This will create some files and install the `@nomiclabs/hardhat-ethers`, `@nomiclabs/hardhat-waffle` plugins, and other necessary packages.
```

Reply:
I think it should be like this, @fvictorio. Shouldn't it?